toolchain/check/convert.cpp

```diff
@@ -707,7 +707,7 @@ static auto CanUseValueOfInitializer(const SemIR::File& sem_ir,
 }
 
 // Returns the non-adapter type that is compatible with the specified type.
-static auto GetCompatibleBaseType(Context& context, SemIR::TypeId type_id)
+static auto GetCompatibleFoundationType(Context& context, SemIR::TypeId type_id)
```

Comment:
I wonder if `GetCompatibleNonAdapterType` would actually work better here -- "compatible" is already supposed to imply that we're looking at adapters (an adapter and its adapted type are compatible with each other).